test/testers.jl

```diff
@@ -711,4 +720,9 @@ struct MySpecialConfig <: RuleConfig{Union{MySpecialTrait}} end
 
         ChainRulesTestUtils.TEST_INFERRED[] = true
     end
+
+    @testset "inference of thunked cotangents" begin
+        @test errors(() -> test_rrule(my_id, 2.0))
+        test_rrule(my_id, 2.0; check_inferred=false)
```

Comment:
test also if that this is fine if we pass `check_thunked_output_tangent=false` ?